napari/layers/_base_layer/model.py

```diff
@@ -123,6 +123,13 @@ def shape(self):
         """
         return self._get_shape()
 
+    @property
+    def range(self):
+        """list of tuple of int: ranges of data for slicing.
```

Comment:
what's the length of each tuple? what does each value in the tuple mean?

Reply:
fixed